gen3/bin/kube-setup-fence.sh

```diff
@@ -25,6 +25,7 @@ setup_audit_sqs() {
 }
 
 gen3 update_config fence-yaml-merge "${GEN3_HOME}/apis_configs/yaml_merge.py"
+gen3 update_config fence-dd-submit-metrics "${GEN3_HOME}/apis_configs/datadog_submit_metrics.py"
```

Comment:
Is this being ran from the adminvm? The adminvm doesn't have any dd agents setup AFAIK :(

Reply:
Or is the python code executed from within the fence pod?

Reply:
Based on some quick reverse engineering , I reckon the `gen3 update_config` creates some configmap and the auxiliary script is executed by the fence container's startup command.

Oh, I'm actually missing a volumeMount in the k8s deployment descriptor...
```
        - name: yaml-merge
          configMap:
            name: "fence-yaml-merge"
        - name: fence-dd-submit-metrics
          configMap:
            name: "fence-dd-submit-metrics"
```

Reply:
Yeah @jawadqur , the script will be executed inside the fence container.